account_asset_management/models/account_move.py

```diff
@@ -59,6 +59,7 @@ def write(self, vals):
         return super().write(vals)
 
     def post(self):
+        super().post()
```

Comment:
Why this change?

Reply:
Because if the assets are created before posting the invoice, they wouldn't have a name and the "code" field of the assets will be empty, but they would have the invoice name.

Reply:
But that is another thing. Please always split these things is separate PRs for having separate debate. I'm also suspecting that the culprit of that is our module `l10n_es_account_invoice_sequence`, not this module.

Reply:
> But that is another thing. Please always split these things is separate PRs for having separate debate. I'm also suspecting that the culprit of that is our module `l10n_es_account_invoice_sequence`, not this module.

I don't think so. As we are working for V13 and l10n_es_account_invoice_sequence does not exist for V13. Might you be talking about V12?

Reply:
Yes, I was talking in v12, that the operation is performed that way. Then it can be of this module, but please put a separate PR with this fix (that doesn't affect the chatter entry), and explain in detail the problem in bot the PR and the commit (`FIX reference empty`) is not enough explanation.

Reply:
Just to know... Would it be OK to create a single PR but several commits for different FIXs and IMPs as long as each was well explained?

Reply:
That can serve, but take into account that if one of the changes generate debate, it will delay the merge of the other features that can be very clear, so you have to balance this for deciding if putting one PR or several.

Reply:
@pedrobaeza as the IMP depends on the FIX, can I do 2 commits?

Reply:
I don't think they depend each other, but you get a "better" chatter entry when the FIX will arrive, but OK.

Reply:
Done